src/main/java/io/lakefs/routerfs/dto/DefaultSchemeTranslation.java

```diff
@@ -0,0 +1,11 @@
+package io.lakefs.routerfs.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Value;
+
+@Value
+@AllArgsConstructor
+public class DefaultSchemeTranslation {
```

Comment:
I find this name not clear.. it is a RawDefaultPathMapping ?

Reply:
It's more of a scheme mapping than a whole prefix mapping. @talSofer, maybe just `DefaultPrefixMapping`?